R/sdf_utils.R

```diff
@@ -0,0 +1,32 @@
+#' Fast cbind for Spark DataFrames
+#'
+#' This is a version of `sdf_bind_cols` that works by zipping
+#' RDDs. From the API docs: "Assumes that the two RDDs have the
+#' *same number of partitions* and the *same number of elements
+#' in each partition* (e.g. one was made through a map on the
+#' other)."
+#'
+#' @keywords internal
+#'
+#' @param ... Spark DataFrames to cbind
+sdf_fast_bind_cols <- function(...) {
```

Comment:
Is it possible for us to validate the requirement above re: two RDDs having same number of partitions + same number of elements in each partition? (Just to future-proof us in case we accidentally use this function where the above doesn't hold)